go-runtime/compile/build.go

```diff
@@ -293,7 +294,49 @@ var scaffoldFuncs = scaffolder.FuncMap{
 	},
 	"valueEnum": func(d schema.Decl) bool {
 		if e, ok := d.(*schema.Enum); ok {
```

Comment:
we can probably simplify these now actually, since we decided Decl.Enum.Type will always be present for value enums and never present for type enums— we could even add a helper, with a comment explaining why the presence of "Type" distinguishes value/type enums, in `enum.go`:
```
// ...
func (e *Enum) IsValueEnum() bool  { return e.Type != nil }
```
and then here just evaluate that

Reply:
Nice!! Adding that now